tests/optimizer/wasm2js.js

```diff
@@ -1,6 +1,12 @@
 // EMSCRIPTEN_START_ASM
 function instantiate(asmLibraryArg, wasmMemory, wasmTable) {
```

Comment:
Is there some way to update this input file along with the expected output when wasm2js changes?   Might be good if there was a corresponding input source file we could use to periodically re-create this?

Reply:
Hmm, sorry, no - this is handwritten, but with some parts copy-pasted from real code to make sure they are "realistic". Perhaps we should add another test that automatically compiles the output. But it would need to grab an intermediate file that we don't emit even in `EMCC_DEBUG=1` atm.